src/views/InteractiveMap/Card.vue

```diff
@@ -181,27 +181,46 @@ export default Vue.extend({
       immediate: true,
       handler() {
         // Reset all modifications when the selected model changes.
-        this.card.reactionAdditions = [];
-        this.card.reactionKnockouts = [];
-        this.card.geneKnockouts = [];
-        this.card.editedBounds = [];
+        this.$store.commit("interactiveMap/updateCard", {
```

Comment:
I would map this mutation into a local method as described in https://vuex.vuejs.org/guide/mutations.html#committing-mutations-in-components

Reply:
That's a nice improvement, thanks :+1: